packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -118,7 +118,8 @@ module.exports = {
           // Point ESLint to our predefined config.
           options: {
             configFile: path.join(__dirname, '../.eslintrc'),
-            useEslintrc: false
+            useEslintrc: false,
+            cache: true
```

Comment:
It's merged,  https://github.com/MoOx/eslint-loader/pull/151#issuecomment-281639113